helpers/any.js

```diff
@@ -15,25 +14,37 @@ const factory = () => {
         const opts = args.pop();
         const predicate = opts.hash;
 
-        if (!_.isEmpty(predicate)) {
+        if (!common.isEmptyObject(predicate)) {
             // With options hash, we check the contents of first argument
-            any = _.any(args[0], predicate);
+            const predicateKeys = Object.keys(predicate);
+            const arg = args[0];
+            if (Array.isArray(arg)) {
+                for (let i = 0; i < arg.length; i++) {
+                    const element = arg[i];
+                    any = common.matchesPredicate(element, predicate, predicateKeys);
```

Comment:
Do you think this will be needed by other helpers? If not, it would be better to keep the function contained in this module.

Reply:
I feel like checking that an object has the same keys as another object can be a useful utility function in itself but there have already been quite a few functions implemented and have only needed it in this instance so maybe better to keep in module.